### PR TITLE
Move guides from Docusaurus to Astro

### PR DESCRIPTION
**Pull request by ncjones**
All existing guides have been migrated to Astro and use the new site structure that is defined at https://www.notion.so/immersve/Docs-Content-Design-858f8fda3f1442ea82f8ddc8f007d792. API references are still using Docusaurus.

### CODE REVIEW

**Comment by ncjones** [comment]
TODO

- [ ] migrate redirects from Docusaurus to Astro
- [ ] fix markdoc tag rendering in production mode

File: imsv-docs-astro/astro.config.mjs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    '/api-reference/accounts-custodial': '/guides/custodial-wallets',
+    '/api-reference/activate-a-card-for-physical-cards-only': '/api-reference/card',
+    '/api-reference/assets': '/api-reference/asset-activities',
+    '/api-reference/block-a-card': '/api-reference/card',
+    '/api-reference/cancel-a-card-asynchronously': '/api-reference/cancel-a-card',
+    '/api-reference/cancel-an-account': '/use-cases/custodial-wallets',
+    '/api-reference/cards-custodial': '/api-reference/card',
+    '/api-reference/claim-a-funding-source-for-an-account': '/api-reference/create-a-funding-source-for-an-account',
+    '/api-reference/close-a-card': '/api-reference/cancel-a-card',
+    '/api-reference/create-a-new-card': '/api-reference/order-card',
```
Comment (catluin) [inline imsv-docs-astro/astro.config.mjs:57]: Question: do we plan to keep deprecated endpoints in the docs for reference?

File: imsv-docs-astro/astro.config.mjs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      ],
+    }),
+    tailwind({
+      applyBaseStyles: false
+    }),
+    markdoc(),
+  ],
+
+  redirects: {
+    '/api-reference/accounts-custodial': '/guides/custodial-wallets',
```
Comment (NicBathgate) [inline imsv-docs-astro/astro.config.mjs:48]: `/guides/custodial-wallets` redirects below to `/use-cases/custodial-wallets`
Could this be changed to go directly to the new URL `/use-cases/custodial-wallets`?

File: imsv-docs-astro/astro.config.mjs (hunk 1 above)
Comment (ncjones) [reply imsv-docs-astro/astro.config.mjs:57]: @catluin I think it depends. These ones were renamed before anyone integrated so we don't need to keep the docs. The redirect is there so links from Slack or Notion etc are preserved.

File: imsv-docs-astro/astro.config.mjs (hunk 2 above)
Comment (ncjones) [reply imsv-docs-astro/astro.config.mjs:48]: @NicBathgate yes, this is a mistake. We should always redirect to the latest resource.

File: imsv-docs-astro/src/styles/tailwind.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  /* make mermaid diagrams readable on dark mode */
+  [data-theme="dark"] p img[src*=".svg"] {
+    filter: invert(0.7);
+  }
+}
```
Comment (NicBathgate) [inline imsv-docs-astro/src/styles/tailwind.css:74]: Do we still need this now mermaid diagrams are all replaced?

**Review by NicBathgate — APPROVED** [review]

File: imsv-docs-astro/src/styles/tailwind.css (hunk 3 above)
Comment (catluin) [reply imsv-docs-astro/src/styles/tailwind.css:74]: We do, the change with mermaid was that we store diagrams in SVG and not mermaid language that is processed by plugin. This CSS snippet styles SVGs

**Pull request merged**